src/mame/drivers/dragon.cpp

```diff
@@ -32,6 +32,7 @@
 #include "bus/coco/coco_ssc.h"
 #include "bus/coco/coco_ram.h"
 #include "bus/coco/coco_orch90.h"
+#include "bus/coco/coco_stecomp.h"
 #include "bus/coco/coco_gmc.h"
```

Comment:
These cartridge headers have gotten mixed up as they’re added.  Please get them back in alphabetical order.